packages/dialtone-vue3/components/pagination/pagination.vue

```diff
@@ -53,8 +52,7 @@
       :aria-label="nextAriaLabel"
       :disabled="isLastPage"
       kind="muted"
-      :importance="isLastPage ? 'primary' : 'clear'"
-      :class="isLastPage ? 'd-fc-black-300 d-bgc-transparent' : 'd-fc-tertiary'"
```

Comment:
You removed this classes that's why the Percy tests detected changes.

Those classes shouldn't be there anyways haven't we removed all utility classes from components?

Can you please update this?

Reply:
ah yeah, it's because the default disabled styling makes it "filled". Added the specific disabled styling to the less file, fixed.